Offroad-iOS/Offroad-iOS/Network/CharacterDetail/DTO/Response/CharacterMotionResponseDTO.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  CharacterMotionResponseDTO.swift
+//  Offroad-iOS
+//
+//  Created by  정지원 on 9/9/24.
+//
+
+import Foundation
+
+struct CharacterMotionResponseDTO: Codable {
+    let message: String
+    let data: CharacterMotionInfo
+}
+
+struct CharacterMotionInfo: Codable {
+    let gainedCharacterMotions: [GainedCharacterMotionList]
+    let notGainedCharacterMotions: [NotGainedCharacterMotionList]
+}
+
+struct GainedCharacterMotionList: Codable {
+    let category: String
+    let characterMotionImageUrl: String
+    let isNewGained: Bool
+}
+
+struct NotGainedCharacterMotionList: Codable {
+    let category: String
+    let characterMotionImageUrl: String
+    let isNewGained: Bool
+}
```

Comment:
P2

두 구조체의 요소가 모두 일치하기 때문에 `CharacterMotionList` 하나로 작성해도 좋을 것 같아요!

```suggestion
struct CharacterMotionInfo: Codable {
    let gainedCharacterMotions: [CharacterMotionList]
    let notGainedCharacterMotions: [CharacterMotionList]
}

struct CharacterMotionList: Codable {
    let category: String
    let characterMotionImageUrl: String
    let isNewGained: Bool
}
```